tools/osv-scanner/main.go

```diff
@@ -37,12 +37,12 @@ func scanDir(query *osv.BatchedQuery, dir string) error {
 
 		if !info.IsDir() {
 			if parser, _ := lockfile.FindParser(path, ""); parser != nil {
-				scanLockfile(query, path)
+				err := scanLockfile(query, path)
 				if err != nil {
 					log.Println("Attempted to scan lockfile but failed: " + path)
 				}
 			}
-			scanSBOMFile(query, path)
+			_ = scanSBOMFile(query, path)
```

Comment:
should we be checking for an error here too instead?

Reply:
I don't think so, because if this succeeds then the dependencies are added to the query array. If this fails, then this is not an SBOM file, so nothing happens.

I'll add a comment here explaining why error checking is skipped.